src/Http/Controllers/AuthorizedAccessTokenController.php

```diff
@@ -4,13 +4,22 @@
 
 use Illuminate\Http\Request;
 use Illuminate\Http\Response;
+use Illuminate\Events\Dispatcher;
 use Laravel\Passport\TokenRepository;
+use Laravel\Passport\Events\AccessTokenRevoked;
 
 class AuthorizedAccessTokenController
 {
     /**
      * The token repository implementation.
      *
+     * @var use Illuminate\Events\Dispatcher;
```

Comment:
```suggestion
     * @var \Illuminate\Events\Dispatcher
```

Reply:
Ready, I changed to use the Dispatcher contratc.